Superbuild/Superbuild.cmake

```diff
@@ -98,6 +102,7 @@ IF(TRAVIS_BUILD)
     SET(DOWNLOAD_TOOLKITS OFF)
     SET(BUILD_HEADLESS ON)
     SET(BUILD_WITH_PYTHON OFF)
+    SET(BUILD_WITH_SCIRUN_DATA OFF)
   ELSE()
     # try building everything with clang!
```

Comment:
@collint8 Turn off the flag for clang too by adding it here, in the ELSE(): 
`SET(BUILD_WITH_SCIRUN_DATA OFF)`